tools/app/app/cards/[key]/page.tsx

```diff
@@ -54,80 +67,86 @@ export default function Page({ params }: { params: { key: string } }) {
     }
   }, [listCard, dispatch]);
 
-  if (error) {
+  if (error || errorCardQuery) {
     let errorMessage = t('unknownError');
     if (error instanceof Error) {
-      errorMessage = (error as Error).message;
+      errorMessage = error.message;
+    }
+    if (errorCardQuery instanceof Error) {
+      errorMessage = errorCardQuery.message;
```

Comment:
This now overwrites if `errorMessage` was already populated by `error.message`. Is query error always more meaningful?

Reply:
I'll leave it like this for now. Soon, cardQuery will replace the other endpoint